utils/run.js

```diff
@@ -76,7 +76,7 @@ async function main() {
   }
 
   execSync(
-    `rm -rf public && mkdir public && echo '<a href="https://zeit.co/new">https://zeit.co/new</a>' > public/index.html`
+    `rm -rf public && mkdir public && echo '<a href="https://zeit.co/new">https://zeit.co/new</a>' > public/index.html && cp -r docs public/docs`
```

Comment:
Perhaps we should remove this script and actually create a `public` directory 😄 
It seems these are just static files so no need for a build script, right?

Reply:
Makes sense 👍(I'll try because I'm not sure it works 🤔)

Reply:
@styfle It doesn't work, see https://zeit.co/lucleray/now/9msxzup6q